tests/conftest.py

```diff
@@ -81,14 +81,21 @@ def fixture_localhost(aiida_localhost):
     return localhost
 
 
+@pytest.fixture(scope="module", autouse=True)
+def aiida_profile(aiida_config, aiida_profile_factory):
+    """Create and load a profile."""
+    with aiida_profile_factory(aiida_config) as profile:
+        yield profile
```

Comment:
Is this still necessary? It's odd that it would be necessary if we're not doing anything like setting up RabbitMQ